modules/integration/src/test/scala/scala/cli/integration/RunScalaJsTestDefinitions.scala

```diff
@@ -291,6 +291,50 @@ trait RunScalaJsTestDefinitions { _: RunTestDefinitions =>
     }
   }
 
+  test("remap imports") {
+    val importmapFile = "importmap.json"
+    val outDir = "out"
+    val fileName = os.rel / "run.scala"
+
+    val inputs = TestInputs(
+      fileName ->
+        s"""//> using jsEsModuleImportMap importmap.json
+          | //> using jsModuleKind es
+          | //> using jsMode fastLinkJS
+          | //> using platform js
+          |
+           |import scala.scalajs.js
+           |import scala.scalajs.js.annotation.JSImport
+           |import scala.scalajs.js.typedarray.Float64Array
+           |
+           |object Foo {
+           |  def main(args: Array[String]): Unit = {
+           |    println(Array(-10.0, 10.0, 10).mkString(", "))
+           |    println(linspace(0, 10, 10).mkString(", "))
+           |  }
+           |}
+           |
+           |@js.native
+           |@JSImport("@stdlib/linspace", JSImport.Default)
+           |object linspace extends js.Object {
+           |  def apply(start: Double, stop: Double, num: Int): Float64Array = js.native
+           |}""".stripMargin,
+      os.rel / importmapFile -> """{"imports": {"@stdlib/linspace": "https://cdn.skypack.dev/@stdlib/linspace"}}""".stripMargin
+    )
+
+    inputs.fromRoot { root =>
+      val absOutDir = root / outDir
+      val outFile = absOutDir / "main.js"
+      os.makeDir.all(absOutDir)
+      os.proc(TestUtil.cli, "--power","package", fileName, "--js", "--js-module-kind", "ESModule", "--js-cli-version", "1.15.0.1", "-o", outFile, "-f", "--js-es-module-import-map", "importmap.json" ).call(cwd = root).out.trim()
+      os.copy.over(outFile, os.home / "Code" / "main.js")
```

Comment:
```
Compiling project (Scala 3.3.1, Scala.js 1.15.0)
Compiled project (Scala 3.3.1, Scala.js 1.15.0)
Downloading https://github.com/virtusLab/scala-js-cli/releases/download/v1.15.0.1/scala-js-ld-aarch64-apple-darwin.gz
Downloaded https://github.com/virtusLab/scala-js-cli/releases/download/v1.15.0.1/scala-js-ld-aarch64-apple-darwin.gz
Wrote /Users/pchabelski/IdeaProjects/scala-cli/out/integration/tmpDirBase.dest/working-dir/run-1335615095/test-2/out/main.js,
```
the package is being called correctly, btw.

Reply:
Yeah, this was me copying the file to a persistent location locally - scala-cli is quite good about tidying up after its test runs, and I wanted to inspect it manually. This will need to go before a merge, clearly.